tests/31sync/06state.pl

```diff
@@ -745,3 +745,158 @@ sub wait_for_event_in_room {
          Future->done(1);
       })
    };
+
+# Test to check that current state events appear in the timeline,
+# even if they were set during a period the user couldn't see
+test "Current state appears in timeline in private history",
```

Comment:
please could you add a reference to a github bug (preferably a synapse one, but an ios one would do)?